Add readme template (#3299)
NuGet/docs.microsoft.com-nuget: docs/nuget-org/package-readme-on-nuget-org.md

* Where and how to leave feedback such as link to the project issues, Twitter, bug tracker, or other platform.
* How to contribute, if applicable.

> For example, package README template looks like this:

Reviewer: ```suggestion
For example, you can start with this package README template:
```

```text
# Package readme title, e.g., display name or title of the package (optional)

Start with a clear and concise description: A brief overview of what your package is and does, also what problem it solves.

## Getting started

Explain how to use your package, provide clear and concise getting started instructions, including any necessary steps.

### Prerequisites

What are specific minimum requirements to use your packages? Consider excluding this section if your package works without any additional setup beyond simple package installation.

## Usage

Examples about how to use your package by providing code snippets/example images, or samples links on GitHub if applicable. 

- Provide sample code using code snippets
- Include screenshots, diagrams, or other visual help users better understand how to use your package

## Additional documentation

> Provide links to more resources: List links such as detailed documentation, tutorial videos, blog posts, or any other relevant documentation to help users get the most out of your package.

Reviewer 2: Based on the [Moq incident](https://snyk.io/blog/moq-package-exfiltrates-user-emails/), should the package owner declare if they have any privacy-related disclosure somewhere in the README file upfront if it really needs to send some information?
